recipe/meta.yaml

```diff
@@ -33,17 +33,36 @@ test:
     - pip
 
 about:
-  home: http://jinja.pocoo.org
+  home: https://palletsprojects.com/p/jinja/
   license: BSD-3-Clause
   license_file: LICENSE.rst
-  summary: An easy to use stand-alone template engine written in pure python.
+  summary: A very fast and expressive template engine.
   description: |
-    Jinja2 is a full featured template engine for Python. It has full unicode
-    support, an optional integrated sandboxed execution environment, widely
-    used and BSD licensed.
-  doc_url: http://jinja.pocoo.org/docs/dev/
-  dev_url: https://github.com/pallets/jinja
-  doc_source_url: https://github.com/pallets/jinja/blob/master/docs/index.rst
+    Jinja is a fast, expressive, extensible templating engine. Special
+    placeholders in the template allow writing code similar to Python
+    syntax. Then the template is passed data to render the final document.
+
+    It includes:
+
+    -   Template inheritance and inclusion.
+    -   Define and import macros within templates.
+    -   HTML templates can use autoescaping to prevent XSS from untrusted
+        user input.
+    -   A sandboxed environment can safely render untrusted templates.
+    -   AsyncIO support for generating templates and calling async
+        functions.
+    -   I18N support with Babel.
+    -   Templates are compiled to optimized Python code just-in-time and
+        cached, or can be compiled ahead-of-time.
+    -   Exceptions point to the correct line in templates to make debugging
+        easier.
+    -   Extensible filters, tests, functions, and even syntax.
+
+    Jinja's philosophy is that while application logic belongs in Python if
+    possible, it shouldn't make the template designer's job difficult by
+    restricting functionality too much.
```

Comment:
Also maybe we should shorten this. Perhaps the first paragraph is sufficient. In general users are likely better off going to the docs for a well reasoned explanation of the project's purpose & objectives.

Reply:
This is similar in length to the Werkzeug and Click descriptions, which were merged. These are taken from the prose at the beginning of each project's readme.

Reply:
Not really. It's just a convention to keep these short. Trying to keep things easy to maintain.

Reply:
@jakirkham and @davidism, I don't have strong feelings about this so work it out between the 2 of you.  I could imagine it would be just as easy to copy several paragraphs as one but the bulleted list is trickier to get update.  But the truth in my experience is that these descriptions don't get updated as part of maintenance regardless of how long or short they are.